### PR TITLE
Icon Sizes Cheatsheet

### LINKED ISSUE #3494: Icon Sizes:
Icon sizes for various Operating Systems.
Windows: 16x16, 24x24, 32x32, 48x48, 256x256
Mac OS X: 16x16, 32x32, 64x64, 128x128, 256x256, 512x512, 1024x1024
Linux: 16x16, 24x24, 48x48, and 96x96
iOS 6: 29x29, 50x50, 57x57, 58x58, 72x72, 100x100, 114x114, 144x144, 1024x1024
iOS 8: 29x29, 40x40, 58x58, 60x60, 76x76, 80x80, 120x120, 152x152, 180x180, 1024x1024
Android L: 24x24, 48x48, 192x192, 512x512
Windows Phone: 62x62, 99x99, 173x173, 200x200

---

IA Page:  http://duck.co/ia/view/icon_sizes 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Nikitaw99

### PR DESCRIPTION
**Pull request by Nikitaw99**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New Icon Sizes Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

screenshot
###### Which issues (if any) does this fix?

Fixes #3494 - Adds the instant answer itself.
###### People to notify (@mention interested parties)

---

Instant Answer Page: http://duck.co/ia/view/icon_sizes

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @Nikitaw99

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Nikitaw99 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by Nikitaw99** [comment]
Amended the commit and force-pushed to fix the error travis gave.

**Comment by Nikitaw99** [comment]
I am taking a look at `https://beta.duckduckgo.com/?q=icon%20size`, however, I am not seeing my cheat sheet anywhere...

**Comment by aiyankovil** [comment]
@Nikitaw99 i would suggest u to change the alias as icon size

File: share/goodie/cheat_sheets/json/icon-sizes.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "id": "icon_sizes_cheat_sheet",
+    "name": "Icon Sizes",
+    "description": "Icon sizes for various Operating Systems.",
+
+    "metadata": {
+        "sourceName": "visualpharm",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/icon-sizes.json]: Lets change source name to "VisualPharm"

**Comment by pjhampton** [comment]
Closing as this looks to have gone stale. 

If you are looking for something more substantial to work on that will have a far-reaching impact on users, please check out the [DuckDuckHack issues table](https://duckduckhack.com/issues).

**Pull request closed without merging**